### PR TITLE
ch02: update hardware wallet instructions

### PR DESCRIPTION
**Pull request by rating89us**

### CODE REVIEW

**Comment by selsta** [comment]
I would replace Bitcoin seed with BIP39 seed, else it might be confusing why a Monero GUI starts talking about Bitcoin seeds.

**Comment by dEBRUYNE-1** [comment]
Merely BIP39 seed may be a bit ambiguous though, perhaps we can add (your private spend key)?

File: en/ch02.md [hunk 1]
```diff
@@ -50,6 +50,34 @@ It has a countdown that allows you to change some settings before your wallet co
...
+
++ The main function of your hardware wallet is to store your Bitcoin seed in a device that never connects to the internet.
++ The Bitcoin seed stored in your device will be used to generate a Monero seed, which will not be displayed to you.
++ The Monero seed inside your device will be used to create two Monero private keys: a view key and a spend key.
++ Your private view key is exported from your hardware wallet device to the Monero GUI wallet.
++ Your private spend key (required to send transactions) never leaves your hardware wallet device.
+
+### Prepare your device
+
+If you are using a Ledger Nano S/X, please make sure:
```
Comment (dEBRUYNE-1) [inline en/ch02.md:63]: Perhaps add that the Ledger Live desktop / manager has to be closed?

Comment (rating89us) [reply en/ch02.md:62]: This is mentioned below, in "Prepare your computer" section

**Comment by rating89us** [comment]
I replaced Bitcoin seed for recovery seed. It's better than BIP-39 (too technical).

**Review by selsta — APPROVED** [review]

**Review by selsta — APPROVED** [review]

**Review by erciccione — APPROVED** [review]
Thanks :)

And thanks @dEBRUYNE-1 and @selsta for the reviews.

**Pull request merged**